aspnetcore/security/authentication/windowsauth.md

```diff
@@ -153,7 +153,7 @@ The [Microsoft.AspNetCore.Authentication.Negotiate](https://www.nuget.org/packag
 > [!NOTE]
 > The Negotiate handler detects if the underlying server supports Windows Authentication natively and if it's enabled. If the server supports Windows Authentication but it's disabled, an error is thrown asking you to enable the server implementation. When Windows Authentication is enabled in the server, the Negotiate handler transparently forwards to it.
 
-Add authentication services by invoking <xref:Microsoft.Extensions.DependencyInjection.AuthenticationServiceCollectionExtensions.AddAuthentication*> and `AddNegotiate` in `Startup.ConfigureServices`:
+Add authentication services by invoking <xref:Microsoft.Extensions.DependencyInjection.AuthenticationServiceCollectionExtensions.AddAuthentication*> and <xref:Microsoft.Extensions.DependencyInjection.NegotiateExtensions.AddNegotiate*> in `Startup.ConfigureServices`:
```

Comment:
For future reference, the asterisk should be encoded as %2A

Reply:
@scottaddie That's a guidance change, correct?

I just want to clarify that I've been following guidance from our prior discussions, so I haven't been doing anything random with these links ...

* `ms.assetid` uses asterisk ...

   ```html
   <meta name="ms.assetid" content="Microsoft.Extensions.DependencyInjection.NegotiateExtensions.AddNegotiate*" />
   ```

* `%2A` was unreliable in the old days 👴 ...... *last year* :smile: when it didn't work in all cases. I know because I tried it in ancient times when I was building the XREF Generator app.

* The asterisk isn't rendered, and the build system doesn't choke and correctly produces ...

   ```
   https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.negotiateextensions.addnegotiate
   ```

Reply:
Here's the official guidance on our internal page:

![image](https://user-images.githubusercontent.com/10702007/75475801-07738d80-595f-11ea-93da-d829e3fa8b74.png)